migrations/20220614130000-update-shares-table-with-columns.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('shares', 'user_id', {
+      type: Sequelize.INTEGER,
+      allowNull: true,
```

Comment:
The `user_id` should be always set. Also, try to use the `uuid` instead.

Reply:
Im need relation with primary key, i cant use uuid, we need in future change user_id with uuid

Reply:
im set allowNull to null because current shares are user_id as null, if not set as null we can have problems with actual rows.